test/test_methods.py

```diff
@@ -149,6 +149,18 @@ def test_load_coordinates(self):
                 for coords, expected in zip(coordinates, objects):
                     self.assertEqual(coords, expected)
 
+    def test_load_coordinates_sci_notation(self):
+
+        # For some datasets that generate coords so close to zero that they end up in scientific notation
+
+        for _ in xrange(NITERS):
```

Comment:
We're always using the same value, so we don't need a for loop here.

Reply:
Fixed